metabench.cmake

```diff
@@ -258,8 +231,8 @@ file(WRITE ${MEASURE_RB_PATH}
 # benchmarks. The template is completed by filling it with the contents
 # of the corresponding JSON file and the required JS libraries.
 ################################################################################
-set(CHART_TEMPLATE_HTML_PATH ${CMAKE_CURRENT_BINARY_DIR}/_metabench/chart_template.html.erb)
-file(WRITE ${CHART_TEMPLATE_HTML_PATH}
+set(CHART_HTML_ERB ${CMAKE_CURRENT_BINARY_DIR}/_metabench/chart.html.erb)
```

Comment:
Oh this one was just for symmetry, as the other global variables were named after the filename along with its extensions. I also removed the `_template` since now it has `.erb` already as an extension.